src/components/Categories.js

```diff
@@ -0,0 +1,11 @@
+import './categories.css';
+
+import React from 'react';
+
+const Cat = () => (
+  <div className="cat-container">
+    <h2>Categories</h2>
+  </div>
+);
+
+export default Cat;
```

Comment:
- [ ] I would recommend renaming this component to `Categories` instead of `Cat` as it is called Categories in the requirement and this will also improve readability and understanding of your codebase.

- [ ] It is required to display `Under construction` here in this component instead of `Categories`.

Reply:
Done